framework/include/iotbus/iotbus_gpio.h

```diff
@@ -233,6 +233,16 @@ int iotbus_gpio_get_edge_mode(iotbus_gpio_context_h dev, iotbus_gpio_edge_e *edg
  */
 int iotbus_gpio_get_drive_mode(iotbus_gpio_context_h dev, iotbus_gpio_drive_e *drive);
 
+/**
+ * @brief registers a signal for current on rising or falling edge of gpio
+ *
+ * @details @b #include <iotbus/iotbus_gpio.h>
+ * @param[in] dev handle of gpio_context
+ * @param[in] edge gpio edge type
+ * @return On success, 0 is returned. On failure, a negative value is returned.
+ * @since Tizen RT v1.x
+ */
+int iotbus_gpio_register_signal(iotbus_gpio_context_h dev, iotbus_gpio_edge_e edge);
```

Comment:
Do you process ACR?

Reply:
Not yet.

Reply:
I will let Jeongin know this. He will do or will ask you this for ACR.